website/pages/blog/auditing-service-account-keys-with-cloudquery.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Auditing Service Account Keys with CloudQuery
+tag: security
+date: 2022/11/04
+description: >-
+  A case against Service Account Keys.
+author: SCKelemen, Alex
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+
+# Intro
+There are several common methods of accessing cloud 
+environments, each with their own risks and uses. Service 
+Account Keys, however, are one of the more dangerous methods.
+Before using service account keys, there are several risks 
+and shortfalls you should consider:
+
+Service Account Keys offer a simple and easy mechanism for 
```

Comment:
Is this a risk or shortfall?  

I would consider moving the Initially paragraph above this to serve as a disclaimer first.